Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/font-weight/index.md

Reviewer: ```suggestion

```

### Values

> This property is specified as one of the following keyword values or a `<number>`:

Reviewer: ```suggestion
his property is specified as a single value from the following list:
```
